### PR TITLE
Setup initial code base server with nodejs and express

### PR DESCRIPTION
**Pull request by ealexisaraujo**
- [ ] Create setup initial code base
- [ ] Config environment variables
- [ ] Install dependencies
- [ ] Setup express

### CODE REVIEW

**Comment by ealexisaraujo** [comment]
> * [ ]  Create setup initial code base
> * [ ]  Config environment variables
> * [ ]  Install dependencies
> * [ ]  Setup express

@hyfi06 Please check this PR

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "healthCarePlatform",
```
Comment (hyfi06) [inline package.json:2]: HALAH Laboratory Platform

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "version": "1.0.0",
```
Comment (hyfi06) [inline package.json:3]: 0.1.0

File: package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "DEBUG=app:* nodemon src/index",
+    "start": "NODE_ENV=production node src/index",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ealexisaraujo/platzi-master-end-game-backend.git"
```
Comment (hyfi06) [inline package.json:13]: :eyes:

File: package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "keywords": [],
```
Comment (hyfi06) [inline package.json:15]: health

File: package.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  "author": "Alexis Araujo <ala1289@gmail.com>",
```
Comment (hyfi06) [inline package.json:16]: :eyes:

File: package.json [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ealexisaraujo/platzi-master-end-game-backend/issues"
```
Comment (hyfi06) [inline package.json:19]: :eyes:

File: package.json [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "homepage": "https://github.com/ealexisaraujo/platzi-master-end-game-backend#readme",
```
Comment (hyfi06) [inline package.json:21]: :eyes:

**Comment by ealexisaraujo** [comment]
@hyfi06 Please check your comments

File: package.json [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+  "keywords": [
+    "health"
+  ],
+  "contributors": [
+    {
+      "name": "Alexis Araujo",
+      "email": "ala1289@gmail.com"
+    },
+    {
+      "name": "Hector Olver",
```
Comment (hyfi06) [inline package.json:24]: Olvera =P

**Pull request merged**